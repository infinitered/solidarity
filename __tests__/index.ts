import baseRuntimeConfiguration from '../src/index'

test('Verify Runtime AND Configuration', () => {
  expect(baseRuntimeConfiguration).toMatchSnapshot()
})

test('ensure build', async () => {
  const result = await baseRuntimeConfiguration()
  expect(result).toMatchSnapshot()
})
