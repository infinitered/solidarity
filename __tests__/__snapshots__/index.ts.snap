// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify Runtime AND Configuration 1`] = `[Function]`;

exports[`ensure build 1`] = `undefined`;
